infrastructure/aws/production/ecs-task-definition-migration.json

```diff
@@ -0,0 +1,51 @@
+{
+    "family": "flagsmith-api-migration",
+    "networkMode": "awsvpc",
+    "executionRoleArn": "arn:aws:iam::084060095745:role/task-exec-role-741a7e3",
+    "taskRoleArn": "arn:aws:iam::084060095745:role/task-exec-role-741a7e3",
+    "containerDefinitions": [
+        {
+            "name": "flagsmith-api-migration",
+            "command": [
+                "migrate"
+            ],
+            "environment": [
+                {
+                    "name": "AWS_REGION",
+                    "value": "eu-west-2"
+                },
+                {
+                    "name": "DJANGO_SETTINGS_MODULE",
+                    "value": "app.settings.production"
+                },
+                {
+                    "name":"IDENTITIES_TABLE_NAME_DYNAMO",
```

Comment:
Do we need these dynamo tables here? As I understand it, this is the _database_ migration task definition so I don't see why we would need them?

Reply:
It's used for both SQL and Identity Migrations